meta-lmp-base/recipes-core/base-files/base-files_%.bbappend

```diff
@@ -1 +1,12 @@
 FILESEXTRAPATHS:prepend:lmp := "${THISDIR}/${PN}:"
+
+SRC_URI:append = " \
+	${@bb.utils.contains('DISTRO_FEATURES', 'luks', 'file://fstab-luks', '', d)} \
+"
+
+do_install:append() {
+	# If LUKS is required, replace standard fstab with fstab-luks
+	if [ "${@bb.utils.contains('DISTRO_FEATURES', 'luks', 'yes', 'no', d)}" = "yes" ]; then
+		install -m 0644 ${WORKDIR}/fstab-luks ${D}${sysconfdir}/fstab
```

Comment:
This and providing should and providing an `fstab-luks` should not be required.
It should be per platform. Not all require this tweak

Another way to mount the new boot part is to use `--part-type bc13c2ff-59e6-4262-a352-b275fd6f7172` in the wic. It will make system mount the part in `/boot`

Reply:
That only works for GPT afaik.